scripts/clean-grant-data

```diff
@@ -34,9 +34,7 @@ const parseDates = data => {
     ];
     dateFields.forEach(dateField => {
         if (data[dateField]) {
-            // We specify several parsing formats here because the source data uses different formats
-            // eg. older data is 2004-04-16 and the rest uses 8/17/15
-            const parsedDate = moment(data[dateField].trim(), ['MM/DD/YYYY', 'YYYY-MM-DD']);
+            let parsedDate = moment(data[dateField].trim().replace('±', '+'));
```

Comment:
Annoyingly, Moment doesn't allow date strings like `2004-04-20T00:00:00±00:00` (which is valid!).

Reply:
(all the newly-formatted data I asked for is using this format)